README.md

```diff
@@ -175,6 +175,17 @@ The plugin already contains some simple elements.
 
 ![The Image collection element](screenshots/image_collection.png)
 
+### Deactivate an available element
+
+Define the overload of a proposed UiElement in your configuration folder, let's say in `config/packages/monsieurbiz_sylius_richeditor_plugin.yaml` as example.
```

Comment:
We could specify what happens to a disabled UI Element.

I would say:
- it's not displayed anymore.
- it's still editable in old contents but you can't add a new one.
- if the element has an alias, the alias is treated the same way.